CMSIS/DSP/Testing/preprocess.py

```diff
@@ -1,18 +1,18 @@
-import argparse
-import TestScripts.NewParser as parse
-import pickle
-
-parser = argparse.ArgumentParser(description='Parse test description')
-
-parser.add_argument('-f', nargs='?',type = str, default=None, help="Test description file path")
-
-parser.add_argument('-o', nargs='?',type = str, default="Output.pickle", help="output file for parsed description")
-
-args = parser.parse_args()
-
-if args.f is not None:
-    p = parse.Parser()
-    # Parse the test description file
-    root = p.parse(args.f)
-    with open(args.o,"wb") as output:
-         pickle.dump(root, output)

```

Comment:
This file has been changed because there was an issue with the new line character

Reply:
@gmiodice Looks good and nice with the pseudo code. Got one naming consideration comment. Channels are usually named as input_ch or output_ch in most other operators.. Maybe you could change just that so it is uniform across files?

Reply:
Since these functions are data layout independent, I add a comment in the doxygen documentation.

Reply:
Since these functions are data layout independent, I added a comment in the doxygen documentation